File: sem-3-labs/dsal/lab4/q4.c
#include <stdio.h>

int tower(int n, char source, char dest, char aux)
{
    static int c = 0;
    if (n == 1)
    {
        printf("\nmove disk 1 from peg %c to peg %c", source, dest);
        c++;
    }
    else
    {
        tower(n - 1, source, aux, dest);
        printf("\nmove disk %d from peg %c to peg %c", n, source, dest);
        c++;
        tower(n - 1, aux, dest, source);
    }
    return c;
}

void main()
{
    int n;
    printf("enter number of discs: ");
    scanf("%d", &n);
    printf("moves are: \n");
    int x = tower(n, 'A', 'B', 'C');
    printf("\nnumber of moves = %d\n", x);
}

File: sem-6-labs/dsl/lab1/q4.py
x = int(input('Enter number 1: '))
y = int(input('Enter number 2: '))
z = int(input('Enter number 3: '))

if x > y and x > z:
    print(x)
elif y > x and y > z:
    print(y)
else:
    print(z)